carlsim/test/stdp.cpp

```diff
@@ -469,7 +469,7 @@ TEST(STDP, ESTDPExpCurve) {
 
 	for (int mode = 0; mode < 2; mode++) {
 		for (int coba = 0; coba < 2; coba++) {
-			for (int offset = -30; offset <= 30; offset += 5) {
```

Comment:
GPU_MODE defaults to CPU_MODE if CARLsim was compiled with nocuda. You are right, it basically runs the same test twice in this case. Maybe I can do that, but I did not want to alter the tests, as travis has not yet tested GPU code or if we test on local machine.